<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QFrame" name="frame_2">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Data type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>Float</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Double</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Loss function</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Mean Squared Error</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Input Delimiter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="focusPolicy">
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Train input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Test input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Save output</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addLayerButton">
          <property name="text">
           <string>Add Layer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="trainButton">
          <property name="text">
           <string>Train</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="testButton">
          <property name="text">
           <string>Test</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>comboBox</zorder>
       <zorder>comboBox</zorder>
       <zorder>label</zorder>
       <zorder>testButton</zorder>
       <zorder>label</zorder>
       <zorder>radioButton</zorder>
       <zorder>label_4</zorder>
       <zorder>toolButton_3</zorder>
       <zorder>trainButton</zorder>
       <zorder>label_2</zorder>
       <zorder>addLayerButton</zorder>
       <zorder>comboBox</zorder>
       <zorder>lineEdit</zorder>
       <zorder>label_3</zorder>
       <zorder>toolButton_2</zorder>
       <zorder>toolButton</zorder>
       <zorder>label_5</zorder>
       <zorder>comboBox_2</zorder>
       <zorder>verticalSpacer</zorder>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="editorScrollArea">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>243</green>
                 <blue>242</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>149</red>
                 <green>154</green>
                 <blue>153</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>243</green>
                 <blue>242</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>243</green>
                 <blue>242</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>149</red>
                 <green>154</green>
                 <blue>153</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>243</green>
                 <blue>242</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="WindowText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Button">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Light">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Midlight">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>239</red>
                 <green>243</green>
                 <blue>242</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Dark">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Mid">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>149</red>
                 <green>154</green>
                 <blue>153</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="BrightText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>255</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ButtonText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>112</red>
                 <green>115</green>
                 <blue>115</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Base">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Window">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="Shadow">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="AlternateBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>224</red>
                 <green>231</green>
                 <blue>230</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipBase">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>255</green>
                 <blue>220</blue>
                </color>
               </brush>
              </colorrole>
              <colorrole role="ToolTipText">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>0</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>332</x>
              <y>0</y>
              <width>16</width>
              <height>425</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QTextEdit" name="textEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Network Output</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
